Makefile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-VERSION := $(shell grep -oP 'version=\K[^[:space:]]+' gradle.properties)
+VERSION := $(shell grep -o '^version=.*-SNAPSHOT' gradle.properties | cut -c9-)
```

Comment:
Is it looking for `SNAPHSOT` in version? How it will work for a release version?

Reply:
fair, let me fix it 😄

Reply:
what about now?